.github/workflows/ci.yml

```diff
@@ -37,5 +37,66 @@ jobs:
       token: ${{ secrets.GITHUB_TOKEN }}
     with:
       version: ${{ needs.version-next.outputs.version-next }}
+      run-frank-till-healthy-enabled: false
 
-  
\ No newline at end of file
+  run-larva-scenarios:
+    runs-on: ubuntu-latest
+    needs:
+      - version-next
+      - ci
+    steps:
+      - name: Checkout
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 #4.1.7
+
+      - name: Download Docker tar
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 #4.1.8
+        with:
+          name: build-docker-image
+
+      - name: Load Docker tar
+        shell: bash
+        run: |
+          docker load --input image.tar
+          docker image ls -a
+
+      - name: Start Application
+        run: |
+          export APP_VERSION="${{ needs.version-next.outputs.version-next }}"
+          docker run -d --name morcoreintegration -p 8080:8080 -e dtap.stage=LOC wearefrank/morcoreintegration:${APP_VERSION}
+
+      - name: Wait for Application to be Healthy
```

Comment:
Add `timeout-minutes: <minutes>` to jobs that do a wait loop. This is to prevent ci/cd from accidentally running infinitely and burning all the expensive ci/cd minutes if anyone ever introduces a bug that causes the loop to not break.